giftless/storage/google_cloud.py

```diff
@@ -1,10 +1,15 @@
+import binascii
+import collections
+import hashlib
 import json
+import logging as log
 import os
-from datetime import datetime, timedelta, timezone
+from datetime import datetime
 from typing import Any, BinaryIO, Dict, Optional
 
 from google.cloud import storage  # type: ignore
 from google.oauth2 import service_account  # type: ignore
+from six.moves.urllib.parse import quote
```

Comment:
Giftless is Python 3.x only, and I see no reason to support legacy Python. So most likely we don't need `six` and can use Python 3.x's `urllib` API directly.

Reply:
What's your suggestion to tackle this? I tried without quote and got a few errors

Reply:
My suggestion is to use `urllib.parse` directly (the Python 3.x API) without using `six` at all because `six` is not needed in Giftless.

Reply:
I just fixed this by replacing `six.moves.urllib.parse` with `urllib.parse`.